src/pykeen/models/unimodal/rgcn.py

```diff
@@ -24,9 +24,66 @@
 class RGCN(
     ERModel[torch.FloatTensor, RelationRepresentation, torch.FloatTensor],
 ):
-    """An implementation of R-GCN from [schlichtkrull2018]_.
+    r"""An implementation of R-GCN from [schlichtkrull2018]_.
 
-    This model uses graph convolutions with relation-specific weights.
+    The \acf{rgcn}~\cite{Schlichtkrull2018} builds on an adapted \ac{gcn} that can be applied on multi-relational data
```

Comment:
The `\ac*` commands should be replaced by their output

Reply:
in general all of the latex style math should be updated to look like the other docs